README.md

```diff
@@ -20,91 +27,98 @@ This code is dual licensed under the MIT license and the OGL (Open Government Li
+ * `smoketest` -- runs tests marked as smoke test against non prod environment
+ * `test` -- runs all tests against non prod environment
+ * `smoketest-prod` -- runs tests marked as smoke test against prod environment
+ * `test-prod` -- runs all tests against prod environment
+ * `check-licenses` -- checks python and node dependencies for GPL licensed code
 
-### Platform setup
+### Testing
+There are some basic end to end tests that run as part of the CI build. These just test _ping and _status and Bundle endpoint are working as expected.   
+To set up variables to run these tests from your machine use the following. You will be asked for your Apigee credentials that you use to log onto Apigee management web page
+```
+export APIGEE_ACCESS_TOKEN=`SSO_LOGIN_URL=https://login.apigee.com get_token`
+export PROXY_NAME=prescriptions-for-patients-internal-dev
+```
+For testing a pull request you can set PROXY_NAME
+```
+export PROXY_NAME=prescriptions-for-patients-pr-<GITHUB PULL REQUEST ID>
+```
 
-As currently defined in your `proxies` folder, your proxies do pretty much nothing.
-Telling Apigee how to connect to your backend requires a *Target Server*, which you should call named `prescriptions-for-patients-target`.
-Our *Target Servers* defined in the [api-management-infrastructure](https://github.com/NHSDigital/api-management-infrastructure) repository.
+You also need to set environment variable SOURCE_COMMIT_ID to the full commit id of the deployed proxy. You can get this using `git log` .   
 
-:bulb: For Sandbox-running environments (`test`) these need to be present for successful deployment but can be set to empty/dummy values.
+Now you can then run the tests using the following 
+```
+make test
+```
+For more information about testing your API see the [API Producer Zone confluence](https://nhsd-confluence.digital.nhs.uk/display/APM/Testing+your+API ).
 
-### Detailed folder walk through
-To get started developing your API use this template repo alongside guidance provided by the [API Producer Zone confluence](https://nhsd-confluence.digital.nhs.uk/display/APM/Deliver+your+API)
 
 #### `/.github`:
 
-This folder contains templates that can be customised for items such as opening pull requests or issues within the repo
+This folder contains workflows and templates related to github
 
-`/.github/workflows`: This folder contains templates for github action workflows such as:
-- `pr-lint.yaml`: This workflow template shows how to link Pull Request's to Jira tickets and runs when a pull request is opened.
-- `continuous-integration.yml`: This workflow template shows how to publish a Github release when pushing to master.
+- `pull_request_template.yml`: Template for pull requests.   
 
+`/.github/workflows`: This folder contains templates for github action workflows such as:
+- `pr-lint.yaml`: This workflow template links Pull Request's to Jira tickets and runs when a pull request is opened.
```

Comment:
Being a pedant about this apostrophe again. Sorry.

Reply:
fixed